plancton/__init__.py

```diff
@@ -135,9 +135,10 @@ def __init__(self, name, pidfile, logdir, rundir, confdir,
     self._force_kill = False
     self._do_main_loop = True
     self._has_image = False
+    self.streamers = []
     self.docker_client = Lazy(lambda: Client(base_url=self._sockpath, version="auto"))
     self.conf = {
-      "influxdb_url"      : None,             # URL to InfluxDB (with #database)
+      "influxdb_url"      : [],               # URL list to InfluxDB (with #database)
```

Comment:
See my comments below: this can easily be made `set()` instead of `[]` if you want.